app/grandchallenge/algorithms/templates/algorithms/data_tables/result_list.html

```diff
@@ -0,0 +1,90 @@
+{% load url %}
+{% load profiles %}
+{% load workstations %}
+{% load guardian_tags %}
+{% load pathlib %}
+{% load humanize %}
+{% load evaluation_extras %}
+
+{{ result.job.created|naturaltime }}
+<td>
```

Comment:
Shouldn't the `<td>` tags be closed?

Reply:
Oh, I see, there is a split later. Should some other identifier be used instead? This confuses PyCharm/VSCode.

Reply:
I agree, this is a bit hacky. I'll think of a cleaner way to do this.

Reply:
I replaced it with a custom `<split/>` tag, it's self closing, so it shouldn't cause any issues with IDEs I'd say.